drunk-uber-driver-simulator/project.godot

```diff
@@ -11,6 +11,7 @@ config_version=5
 [application]
 
 config/name="Mini-jame-gam-26"
+run/main_scene="res://Scenes/MainMenu.tscn"
```

Comment:
Lmk if I should remove this, it might be inconvenient for dev work on the main gameplay